iml-gui/crate/src/page/filesystem.rs

```diff
@@ -79,10 +85,45 @@ pub fn init(cache: &ArcCache, model: &mut Model, orders: &mut impl Orders<Msg, G
     stratagem::init(cache, &model.stratagem, &mut orders.proxy(Msg::Stratagem));
 
     orders.send_msg(Msg::FetchStats);
+
+    orders.send_msg(Msg::FetchMountCommand);
+}
+
+async fn delay_and_fetch_mount_command(delay: u32) -> Result<Msg, Msg> {
```

Comment:
use `sleep_with_handle` instead.

Reply:
Sure. When we upgrade to 0.7 (or even 0.8) we can use streams on the orders object :)

Reply:
yup